### DISCUSSION TITLE
execute function at the end of the timer

### DISCUSSION THREAD

**Discussion by domedfd** [vydimitrov/react-countdown-circle-timer#145] [Q&A] [answered]
Hello, I want to execute myFunction () in the last 3 seconds of the timer, when I put the onComplete property it is executed at the beginning of the timer, how can I do?

**Comment by vydimitrov** [comment, marked as answer]
Hey, in the `children` prop you have access to the remaining time, when it is less than 3 seconds just fire your event.

**Reply by domedfd** [reply]
Hellooo! thanks for answering me!
Try to do the following, but call the function several times and the application crashes, here I did an example with the console.log

remark: I am using for react-native

![imagem](https://user-images.githubusercontent.com/8238664/120403074-703f7780-c311-11eb-9a39-56ca4d43672c.png)

**Reply by vydimitrov** [reply]
I am having hard time reading your code, I'd recommend using `prettier`. Where do you close the `if` condition? Can you create ExpoSnack with your code reproducing the issue and I will take a look on it?
